docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+## Data-driven visualizations
+
+### What is a ramp?
+
+[`ramp`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsramp) is a special CARTO VL expression that outputs values based on an input. Depending on the type of the input the matching will be performed in different ways:
+- One-to-one mapping is performed when the number of possible categories in the input matches the number of values. For example, `ramp(buckets($winner, ["Conservative Party", "Labour Party"]), [blue, red])` will set conservatives blue,
+- Interpolation is performed otherwise, this allows to create intermediate values automatically. For example: `color: ramp($population_density, [green, yellow, red])` will assign the color green to the features with low population density and red to the ones with a high population density. Intermediate population densities will receive the interpolation between green, yellow and red based on how close its value is to the lowest and highest values in the dataset.
+
+It's easy to create choropleth maps by using `ramp` with colors as the values. However, `ramp` values don't need to be colors, allowing creating different and richer types of maps like bubble-maps. But, for simplicity's sake, we will stick to colors until the [Ramp Values section](#Ramp-values).
+
+### Ramp inputs
```

Comment:
Maybe it would be better to remove this 3rd level here, adding its description to "What is a ramp", and then promote #### like "Numerical properties" and "Categorical properties" to this 3rd level, maybe even renaming them to something like "Ramp for numerical props", "Ramp for categorical props"

Reply:
I don't understand, what would you do with the content of "what is a rmap"?

Reply:
I would keep it and include there the first two paragraphs of "Ramp inputs", to make the introduction a little longer. Then remove "Ramp inputs" title and push up its children one level up:

- What is a ramp?
- Ramps for numerical properties
    - Showing unclassified numerical data
    - ...

Reply:
:+1: